launch/awl_description.launch.py

```diff
@@ -0,0 +1,43 @@
+import launch
```

Comment:
Can you please as a README to the launch folder that briefly describes what each launch file will do.